mlflow/recipes/utils/wrapped_recipe_model.py

```diff
@@ -0,0 +1,34 @@
+from mlflow.pyfunc import PythonModel
+import pandas as pd
+import numpy as np
+
+
+class WrappedRecipeModel(PythonModel):
+    def __init__(self, classifier, predict_scores_for_all_classes=True, prefix="predicted_"):
+        super().__init__()
+        self._classifier = classifier
```

Comment:
The `classifier` shouldn't be part of the constructor, since this will pickle the classifier along with the model class (this may work okay for scikit-learn but doesn't work for many ML model types that can't be pickled)

Instead, we should pass in a *reference* to the classifier at model save time via the `artifacts` parameter to `mlflow.pyfunc.log_model()` and then load it via the `PythonModel.load_context()` method: Here's an example -  https://mlflow.org/docs/latest/models.html#example-saving-an-xgboost-model-in-mlflow-format